sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -396,6 +396,7 @@ object GpuOverrides {
 
   def isSupportedType(dataType: DataType): Boolean = dataType match {
       case BooleanType => true
+      case BinaryType => true
```

Comment:
I don't think we can generically support this yet.  We need to test all existing operators that might be able to support binary types before we can add this in generically everywhere.  Then we need to either explicitly allow this type for each operator that we know works with it, or if there are enough that it works for we need to explicitly disallow it for all of the operators that it does not work on.

Can Parquet and Orc read/write this yet?
Can we shuffle this yet?
Can we pull it back to the CPU or send it to the GPU?
Does it work as a part of a filter or a join?
Can we get the length of it (GpuLength)?